### DISCUSSION TITLE
Use Card component in Astro

### DISCUSSION THREAD

**Discussion by tecsoc** [heroui-inc/heroui#1768] [Help]
Related: #1741 

I think that want to use nextui in astro project.

I read [document](https://nextui.org/docs/frameworks/astro) and installed it.
I was able to use simple component (ex. Button).

Also through I did copy and paste document [Usage example](https://nextui.org/docs/components/card#usage), but I couldn't use Card component.
I got this error.

```useCardContext: `context` is undefined. Seems you forgot to wrap component within <Card />```

How to fix it ?
I saw using Nano Stores instead context in Astro. 
I hope to be able to use in Astro and got your help.

**Comment by delaudio** [comment]
@tecsoc you can solve it [this way](https://github.com/nextui-org/nextui/discussions/1741#discussioncomment-7876568) (#1741)
